astropy/stats/bayesian_blocks.py

```diff
@@ -431,16 +443,19 @@ class RegularEvents(FitnessFunc):
     dt : float
         tick rate for data
     p0 : float (optional)
-        False alarm probability, used to compute the prior on N
-        (see eq. 21 of Scargle 2012).  Default prior is for p0 = 0.
+        False alarm probability, used to compute the prior on N_blocks
+        (see eq. 21 of Scargle 2012).  Default prior is for p0 = 0.05
```

Comment:
No need for the ``Default prior is for p0 = 0.05``, because ``p0=0.05`` already shows up in the docs automatically (and in ipython if you do ``RegularEvents?``).  Same thing a few other places below.